libraries/AP_HAL_REVOMINI/hardware/STM32F4xx_DSP_StdPeriph_Lib_V1.1.0/Libraries/CMSIS/Device/ST/STM32F4xx/Include/system_stm32f4xx.h

```diff
@@ -0,0 +1,105 @@
+/**
+  ******************************************************************************
+  * @file    system_stm32f4xx.h
+  * @author  MCD Application Team
+  * @version V1.1.0
+  * @date    11-January-2013
+  * @brief   CMSIS Cortex-M4 Device System Source File for STM32F4xx devices.       
+  ******************************************************************************  
+  * @attention
+  *
+  * <h2><center>&copy; COPYRIGHT 2013 STMicroelectronics</center></h2>
+  *
+  * Licensed under MCD-ST Liberty SW License Agreement V2, (the "License");
+  * You may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at:
+  *
+  *        http://www.st.com/software_license_agreement_liberty_v2
```

Comment:
pointer to the " MCD-ST Liberty SW License Agreement V2" for discussion.

Reply:
ChibiOS use CMSIS and is present on the modules, use that or in separate module for revomini hal.

https://github.com/ArduPilot/ChibiOS/blob/fd01e98bc7aa657f883d0d0860d0b47f678ca098/os/common/ext/CMSIS/ST/STM32F4xx/system_stm32f4xx.h